tests/session/transaction-integration_error-004.phpt

```diff
@@ -0,0 +1,70 @@
+--TEST--
+MongoDB\Driver\Session: Setting per-op readConcern or writeConcern in transaction (executeReadWriteCommand)
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto(); ?>
+<?php skip_if_not_replica_set(); ?>
+<?php skip_if_server_version('<', '4.0'); ?>
+<?php skip_if_not_clean(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI);
+
+/* Create collections as that can't be (automatically) done in a transaction */
+$manager->executeCommand(
+    DATABASE_NAME,
+    new \MongoDB\Driver\Command([ 'create' => COLLECTION_NAME ]),
+    [ 'writeConcern' => new \MongoDB\Driver\WriteConcern( \MongoDB\Driver\WriteConcern::MAJORITY ) ]
+);
+
+
+/* Do the transaction */
+$session = $manager->startSession();
+
+$session->startTransaction( [
+    'readConcern' => new \MongoDB\Driver\ReadConcern( "snapshot" ),
+    'writeConcern' => new \MongoDB\Driver\WriteConcern( \MongoDB\Driver\WriteConcern::MAJORITY )
+] );
+
+echo throws(function() use ($manager, $session) {
+    $cmd = new \MongoDB\Driver\Command( [
+        'update' => COLLECTION_NAME,
+        'updates' => [ [ 'q' => [ 'employee' => 3 ], 'u' => [ '$set' => [ 'status' => 'Inactive' ] ] ] ]
```

Comment:
I'm not sure this is the best command, as [`update`](https://docs.mongodb.com/manual/reference/command/update/) doesn't actually support a `readConcern` option. Based on the list of supported commands in [the spec](https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#read-concern) it looks like `readConcern` is not an option for any commands that write (e.g. `aggregate` supports `readConcern` but not when `$out` is used). In that case, you'd do better to simply execute a proper read command here.

I realize this doesn't matter from an integration standpoint, as the driver will throw long before we send this command to the server, but it would make for a more correct example in the test if anyone reads this later.

Reply:
I think you added this to the wrong section, as we're really just checking a ``writeConcern`` here. The comment should have belonged to the previous [``throws`` as there we use a ``readConcern``](https://github.com/mongodb/mongo-php-driver/pull/902/files#diff-01a050f3d755cc3e7644ad65afc22ee2R41), which I've updated.

Reply:
> I think you added this to the wrong section

Indeed. This was intended for the previous section.